src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -23,18 +23,34 @@
  */
 public class OSIsoftServer {
 
+  /** Boolean to represent when the last sent message was succesfully sent. */
   private static boolean connected = true;
 
+  /** Ewon error code representing a link error */
   public static final int LINK_ERROR = 32601;
+
+  /** Ewon error code representing a send error */
   public static final int SEND_ERROR = 32603;
+
+  /** Ewon error code representing a JSON error */
   public static final int JSON_ERROR = -100;
 
+  /** Ewon error code representing no error has occured */
   public static final int NO_ERROR = 0;
+
+  /** Ewon error code representing a Ewon error */
   public static final int EWON_ERROR = 1;
+
+  /** Ewon error code representing an authentication error */
   public static final int AUTH_ERROR = 2;

```

Comment:
https://github.com/hms-networks/sc-ewon-flexy-extensions-lib/blob/4df3732be4aec313507aae6fed445d0fb1592514/src/main/java/com/hms_networks/americas/sc/extensions/system/http/SCHttpUtility.java#L31

Reply:
@oliver-walker-hms  and @alexjhawk does it make sense to put the error codes defined here that aren't in the extensions library in as well. Its hard enough to figure out what the errors are from the standard documentation so i could see them being useful if defined in that location rather than here.